### ISSUE TITLE
may 1

### ISSUE THREAD

**Issue by hillarysanders** [hillarysanders/notes#23]
Last friday:
- [x] was having memory leakage in python data fetch; fixed. Fetched new data with all 4096 shorthashes.
- [x] tested out best convnet model on sophos benign data
- [x] make slides explaining a feed forward pass of a neural network :)!!! Nice!
 
TODO:
minis:
- [x] make combined sophos, VT ROC curve.
    - [x] compare FPR rate in VT (training dataset, validation set) data to FPR rate in Sophos benign data on final fitted convnet.
- [ ] learn about Lambdas on AWS,  spin one up to create a tiny working API
   - [ ] create instructions or scripts to create one of these
    - [ ] fix permissions
    - [ ] add in a working model (e.g. convnet), test out
- [ ] encoding of URL characters
